field_test.go

```diff
@@ -178,14 +178,26 @@ func TestIntsField(t *testing.T) {
 	assertFieldJSON(t, `"foo":[]`, Object("foo", []int{}))
 	assertFieldJSON(t, `"foo":[1]`, Object("foo", []int{1}))
 	assertFieldJSON(t, `"foo":[1,2,3]`, Object("foo", []int{1, 2, 3}))
+
+	assertFieldJSON(t, `"foo":[]`, Ints("foo", []int{}))
+	assertFieldJSON(t, `"foo":[1]`, Ints("foo", []int{1}))
+	assertFieldJSON(t, `"foo":[1,2,3]`, Ints("foo", []int{1, 2, 3}))
+
 	assertCanBeReused(t, Object("foo", []int{1, 2, 3}))
+	assertCanBeReused(t, Ints("foo", []int{1, 2, 3}))
 }
 
 func TestStringsField(t *testing.T) {
 	assertFieldJSON(t, `"foo":[]`, Object("foo", []string{}))
 	assertFieldJSON(t, `"foo":["bar 1"]`, Object("foo", []string{"bar 1"}))
 	assertFieldJSON(t, `"foo":["bar 1","bar 2","bar 3"]`, Object("foo", []string{"bar 1", "bar 2", "bar 3"}))
+
+	assertFieldJSON(t, `"foo":[]`, Strings("foo", []string{}))
+	assertFieldJSON(t, `"foo":["bar 1"]`, Strings("foo", []string{"bar 1"}))
+	assertFieldJSON(t, `"foo":["bar 1","bar 2","bar 3"]`, Strings("foo", []string{"bar 1", "bar 2", "bar 3"}))
+
 	assertCanBeReused(t, Object("foo", []string{"bar 1", "bar 2", "bar 3"}))
+	assertCanBeReused(t, Strings("foo", []string{"bar 1", "bar 2", "bar 3"}))
```

Comment:
Good suggestion and done.  Note that the TestStringsFieldQuick currently fails if it is passed a string containing "&".  Here's some test output with an input of []string{"&"}:

mis-match
   sobj: "foo":["\u0026"]
  sstrs: "foo":["&"]

Akshay or I will look for a solution to this.